docs/docs/guides/web3_utils_migration_guide.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 6
```

Comment:
```suggestion
sidebar_position: 10
```

Reply:
Currently there are 9 sections in migration guide so we should put utils section at end.